data/test_domains/duplicate_intents.yml

```diff
@@ -26,3 +26,7 @@ actions:
   - utter_default
   - utter_greet
   - utter_goodbye
+
+config:
```

Comment:
I think this should be renamed to something more descriptive (because we already have a config file and the domain is a configuration in general):
- we could rename to `sessions` 
- we could remove the dict and make the keys top level

Reply:
renaming doesn't work, because of the `store_entities_as_slots`. We could add another key or make them top level. Any preferences? Adding another key is a bit easier because this wouldn't change things for `store_entities_as_slots`